test/e2e/mxGraph.model.test.ts

```diff
@@ -72,6 +81,25 @@ export interface ExpectedStartEventModelElement extends ExpectedEventModelElemen
   isInterrupting?: boolean;
 }
 
+const bpmnVisualization = new BpmnVisualization(null);
+
+expect.extend({
+  toBeCell(cellId) {
+    const cell = bpmnVisualization.graph.model.getCell(cellId);
+    if (cell) {
+      return {
+        message: () => `Expected cell with id '${cellId}' NOT to be defined`,
```

Comment:
:question: if this more accurate, I guess that when the expect pass, we have to provide what is ok
```suggestion
        message: () => `Expected cell with id '${cellId}' is defined`,
```

Reply:
The message must be negative. See the Jest exemple: https://jestjs.io/docs/en/expect
I run the test with a error, and the right message is displayed.

Reply:
Fair enough, I hadn't found the Jest expect convention.
The Jest doc is clear about that and the new messages are fine.